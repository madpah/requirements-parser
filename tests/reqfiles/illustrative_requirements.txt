-e git+https://github.com/davidfischer/requirements-parser.git#egg=requirements
Django >=1.5, <1.6
numpy
DocParser [PDF]
