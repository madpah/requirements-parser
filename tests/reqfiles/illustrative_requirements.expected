[
 {
  "specifier": false, 
  "local_file": false, 
  "name": "requirements", 
  "editable": true, 
  "uri": "git+https://github.com/davidfischer/requirements-parser.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git+https://github.com/davidfischer/requirements-parser.git#egg=requirements", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": true, 
  "local_file": false, 
  "name": "Django", 
  "editable": false, 
  "uri": null, 
  "extras": [], 
  "vcs": null, 
  "path": null, 
  "line": "Django >=1.5, <1.6", 
  "specs": [
   [
    ">=", 
    "1.5"
   ], 
   [
    "<", 
    "1.6"
   ]
  ], 
  "revision": null
 }, 
 {
  "specifier": true, 
  "local_file": false, 
  "name": "numpy", 
  "editable": false, 
  "uri": null, 
  "extras": [], 
  "vcs": null, 
  "path": null, 
  "line": "numpy", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": true, 
  "local_file": false, 
  "name": "DocParser", 
  "editable": false, 
  "uri": null, 
  "extras": [
   "pdf"
  ], 
  "vcs": null, 
  "path": null, 
  "line": "DocParser [PDF]", 
  "specs": [], 
  "revision": null
 }
]